packages/avatars/src/_state.css

```diff
@@ -2,46 +2,28 @@
 
 /* stylelint-disable no-unsupported-browser-features */
 :root {
-  --zd-avatar-active-background-color: var(--zd-color-malibu);
-  --zd-avatar-in-background-color: var(--zd-color-apple-green);
-  --zd-avatar-out-background-color: var(--zd-color-gainsboro);
-  --zd-avatar-out-filter-url: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="grayscale"><feColorMatrix values=".3333 .3333 .3333 0 0 .3333 .3333 .3333 0 0 .3333 .3333 .3333 0 0 0 0 0 1 0"/></filter></svg>#grayscale');
-  --zd-avatar-out-opacity: .5;
+  --zd-avatar-disabled-background-color: var(--zd-color-grey-300);
+  --zd-avatar-disabled-filter-url: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg"><filter id="grayscale"><feColorMatrix values=".3333 .3333 .3333 0 0 .3333 .3333 .3333 0 0 .3333 .3333 .3333 0 0 0 0 0 1 0"/></filter></svg>#grayscale');
```

Comment:
Can you use the svg-load function here?

Reply:
`svg-load` requires a file (on a configured path). I don't think it's worth it to get fancy here.